Bangazon/Models/PaymentType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Bangazon.Models
+{
+    public class PaymentType
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string AcctNumber { get; set; }
```

Comment:
The data type for `AcctNumber` is an `int` in the database, so should be one in the model.

Reply:
I have made the proper changes to this